go/interpreter/lfvm/interpreter.go

```diff
@@ -247,7 +247,7 @@ func steps(c *context, oneStepOnly bool) {
 		}
 
 		// Consume static gas price for instruction before execution
-		if !c.useGas(staticGasPrices.get(op)) {
+		if c.useGas(staticGasPrices.get(op)) != nil {
```

Comment:
Can you run a benchmark like fib before and after this change? Just to be safe that replacing a boolean return value with a interface return type is not adding significant overhead here by losing some compiler optimizations.

Reply:
```
         │  old.benc   │              new.benc              │
         │   sec/op    │   sec/op     vs base               │
Fib10-16   88.84µ ± 1%   88.17µ ± 2%  -0.76% (p=0.019 n=50)
```
I think we are safe.